### PR TITLE
feat(R-CodeSnippets) : Adding R examples for PA V3 Calculations

### PR DESCRIPTION
**Pull request by vamshimupparaju**
Adding R examples which are used to get PA V3 Calculations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,9 +36,5 @@ setwd("/path/to/analyticsapi-engines-r-sdk")
 Sys.setenv("ANALYTICS_API_URL" = "https://api.factset.com")
 Sys.setenv("ANALYTICS_API_USERNAME_SERIAL" = "<username-serial>")
 Sys.setenv("ANALYTICS_API_PASSWORD" = "<apikey>")
-```
-
-## Contributing
-
-* Files in [auto-generated-sdk](auto-generated-sdk) directory are auto-generated and should not be manually edited here. Refer [Analytics API Engines SDK Generator](https://github.com/factset/analyticsapi-engines-sdk-generator) for instructions on how to modify these files.
-* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
+Sys.setenv("CLIENT_PROXY_URL" = "<proxyurl>")PROXY_URL
```
Comment (saigiridhar21) [inline README.md]: Very minor: Is there a typo mistake in this line?

**Review by saigiridhar21 — APPROVED** [review]
Looks good to me

File: README.md [hunk 2]
```diff
@@ -1,61 +1,33 @@
...
-# Analytics API Engines R SDK
+# Analytics API V3 R Programming Code Snippets
 
-[![build](https://img.shields.io/github/workflow/status/factset/analyticsapi-engines-r-sdk/CI)](https://github.com/factset/analyticsapi-engines-r-sdk/actions?query=workflow%3ACI)
-[![cran](https://img.shields.io/cran/v/factset.analyticsapi.engines)](https://cran.r-project.org/web/packages/factset.analyticsapi.engines)
-![API version](https://img.shields.io/badge/API-v2-blue)
-[![Apache-2 license](https://img.shields.io/badge/license-Apache2-brightgreen.svg)](https://www.apache.org/licenses/LICENSE-2.0)
-
-Use this library to integrate with FactSet's Analytics APIs. Below APIs are supported by this SDK.
+Use this examples to work with FactSet's Analytics APIs. Below APIs can be found in this Repository.
```
Comment (saigiridhar21) [inline README.md]: `To integrate with FactSet's Analytics APIs. Code snippets are available for below APIs.`

File: README.md (hunk 1 above)
Comment (vamshimupparaju) [reply README.md]: it is a type mistake cleaned up

File: README.md (hunk 2 above)
Comment (vamshimupparaju) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -1,72 +1,38 @@
...
 Sys.setenv("ANALYTICS_API_URL" = "https://api.factset.com")
 Sys.setenv("ANALYTICS_API_USERNAME_SERIAL" = "<username-serial>")
 Sys.setenv("ANALYTICS_API_PASSWORD" = "<apikey>")
-```
-
-## Contributing
-
-* Files in [auto-generated-sdk](auto-generated-sdk) directory are auto-generated and should not be manually edited here. Refer [Analytics API Engines SDK Generator](https://github.com/factset/analyticsapi-engines-sdk-generator) for instructions on how to modify these files.
-* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
+Sys.setenv("CLIENT_PROXY_URL" = "<proxyurl>")
```
Comment (saigiridhar21) [inline README.md]: Let's remove the `CLIENT_PROXY_URL` as it is an optional one.

File: README.md [hunk 4]
```diff
@@ -1,72 +1,38 @@
...
 ```r
 Sys.setenv("ANALYTICS_API_URL" = "https://api.factset.com")
 Sys.setenv("ANALYTICS_API_USERNAME_SERIAL" = "<username-serial>")
 Sys.setenv("ANALYTICS_API_PASSWORD" = "<apikey>")
-```
-
-## Contributing
-
-* Files in [auto-generated-sdk](auto-generated-sdk) directory are auto-generated and should not be manually edited here. Refer [Analytics API Engines SDK Generator](https://github.com/factset/analyticsapi-engines-sdk-generator) for instructions on how to modify these files.
-* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
```
Comment (saigiridhar21) [inline README.md:72]: Can you please add this line back?

**Review by saigiridhar21 — APPROVED** [review]

File: examples/PAV3SingleCalculation.R [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+library(factset.protobuf.stach.v2)
+library(stringr)
+
+# Reading environment variables
+
+host <- Sys.getenv("FACTSET_HOST")
+username <- Sys.getenv("FACTSET_USERNAME")
+password <- Sys.getenv("FACTSET_API_KEY")
+#proxyUrl <-
+#Sys.getenv("CLIENT_PROXY_URL") # To Set Up Proxy Configuration
```
Comment (Ramanth) [inline examples/PAV3SingleCalculation.R]: minor: correct the alignment

File: examples/PAV3SingleCalculation.R [hunk 6]
```diff
@@ -0,0 +1,327 @@
...
+    "POST" = apiResponse <- RETRY(
+      "POST",
+      url,
+      # use_proxy(url = proxyUrl), # uncomment this to set up proxy url
+      authenticate(username, password),
+      add_headers("Accept" = "application/json", customHeaders),
+      content_type("application/json"),
+      pause_min = 2,
+      times = maxCalls,
+      # Maximum number of requests to attempt
```
Comment (Ramanth) [inline examples/PAV3SingleCalculation.R:77]: lets move this comment to above line

**Review by Ramanth — APPROVED** [review]

File: examples/PAV3SingleCalculation.R (hunk 5 above)
Comment (vamshimupparaju) [reply examples/PAV3SingleCalculation.R]: done

File: README.md (hunk 3 above)
Comment (vamshimupparaju) [reply README.md]: done

File: examples/PAV3SingleCalculation.R (hunk 6 above)
Comment (vamshimupparaju) [reply examples/PAV3SingleCalculation.R:77]: done

**Review by ksreeramoj — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -69,4 +39,4 @@ Sys.setenv("ANALYTICS_API_PASSWORD" = "<apikey>")
 ## Contributing
 
 * Files in [auto-generated-sdk](auto-generated-sdk) directory are auto-generated and should not be manually edited here. Refer [Analytics API Engines SDK Generator](https://github.com/factset/analyticsapi-engines-sdk-generator) for instructions on how to modify these files.
-* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
+* Projects [examples](examples) and [tests](tests) are open to enhancements and bug fixes. Please create a pull requests with the proposed changes.
```
Comment (saigiridhar21) [inline README.md]: Let's remove the line 41 and in line 42 talk about only examples.

**Review by saigiridhar21 — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (vamshimupparaju) [reply README.md]: done

**Pull request merged**